docs/source/manual/jaxrs.rst

```diff
@@ -26,6 +26,26 @@ To include the correlation id in the catalina.out file using log4j, update the l
 .. |usageLink| replace:: More on this later
 .. _usageLink: `Usage`_
 
+PATCH
+--------------
+
+`PATCH is a HTTP verb (similar to GET, POST, DELETE, PUT) to push partial changes to the REST resource. JAX-RS by default doesn't support PATCH hence Beadledom adds the support for PATCH explicitly.
```

Comment:
link to wiki on brian's version of rst [f388ee2](https://github.com/cerner/beadledom/pull/36/commits/f388ee2f8c06434cc427eafd2717d1f38ba67d1d)